src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6256,6 +6261,10 @@ private ScanTypeFlags ScanNonArrayType(out SyntaxToken lastTokenOfType)
             return result;
         }
 
+        /// <summary>
+        /// Returns TupleType when a possible tuple type is found.
+        /// Note that this is not MustBeType, so that the caller can consider deconstruction syntaxes.
+        /// </summary>
         private ScanTypeFlags ScanTupleType(out SyntaxToken lastTokenOfType)
```

Comment:
You should mention that the opening paren of the type will have already been consumed by the caller.